gamla/functional_generic.py

```diff
@@ -649,3 +649,17 @@ def groupby(
         ),
         valmap(tuple),
     )
+
+
+def countby(f: Callable) -> Dict[Any, int]:
```

Comment:
I think most of our functions are called `x_by` (with an underscore) e.g. `sort_by`